packages/resource-deployment/scripts/role-assign-for-sp.sh

```diff
@@ -35,19 +35,19 @@ echo "Granting role '$role' to the resource group '$resourceGroupName' for servi
 end=$((SECONDS + 300))
 printf " - Running .."
 while [ $SECONDS -le $end ]; do
-    az role assignment create --role "$role" --resource-group "$resourceGroupName" --assignee-object-id "$principalId" 1>/dev/null
-    principalIdResponse=$(az role assignment list --resource-group "$resourceGroupName" --assignee "$principalId" --query "[?principalId=='$principalId'].principalId" -o tsv)
-
-    if [[ $principalIdResponse == "$principalId" ]]; then
-        break
-    fi
-
+    response=$(az role assignment create --role "$role" --resource-group "$resourceGroupName" --assignee-object-id "$principalId" --query "roleDefinitionId") || true
+
+    if [[ -n $response ]]; then	
+        break	
+    else	
+        echo "Retry count - $SECONDS."	
```

Comment:
either print retry count (like we had before. in my old commit. or just printf '.' this line will print seconds instead of retry count.